pages/common/nyxt.md

```diff
@@ -0,0 +1,20 @@
+# nyxt
+
+> Nyxt is a keyboard-driven web browser for power users.
+> More information: <https://nyxt.atlas.engineer>.
+
+- Print system information:
```

Comment:
This doesn't really feel like it's the most important flag. Maybe we should move it to the bottom?